libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/media/AudioDetails.kt

```diff
@@ -16,9 +16,10 @@
 
 package io.element.android.libraries.matrix.api.media
 
+import kotlinx.collections.immutable.ImmutableList
 import kotlin.time.Duration
 
 data class AudioDetails(
```

Comment:
Why no @Immutable here ?

Reply:
Both `Duration` and the `ImmutableList` used inside are vals and stable, so the data class is automatically considered immutable too:

```
stable class AudioDetails {
  stable val duration: Duration
  stable val waveform: ImmutableList<Float>
  <runtime stability> = 
}
```

Reply:
So why has it been added on `AudioInfo` ?

Reply:
Oh, it wasn't needed. It seems like I got some false positive in the compose compiler reports. I removed the annotation and now it's still displaying it's stable. Nice catch!